src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -129,6 +130,53 @@ public void testGetTableData() throws SQLException {
         assertArrayEquals(expected, actual);
     }
 
+    @Test
+    public void testDataSetToString() {
+        DataSet dataSet = createDataSet(new String[]{"1", "name1", "25"});
+        assertEquals("'1','name1','25'", dataSet.toString());
+    }
+
+    @Test
+    public void testInsertWhenSuccess() throws SQLException {
+        DataSet dataSet = createDataSet(new String[]{"111", "someName", "25"});
+        String tableName = "table_name";
+        createTableWithData(tableName);
+        databaseManager.connect(dbName, USER, PASSWORD);
+        boolean actual = databaseManager.insert(tableName, dataSet);
+        executeQuery(String.format("DROP TABLE IF EXISTS %s", tableName));
+        assertTrue(actual);
+    }
+
+    @Test
+    public void testInsertWhenNoTable() {
+        DataSet dataSet = createDataSet(new String[]{"1", "name1", "25"});
+        databaseManager.connect(dbName, USER, PASSWORD);
+        assertFalse(databaseManager.insert("noTable", dataSet));
+    }
+
+    @Test
+    public void testInsertWhenExtraColumnsCount() throws SQLException {
+
+        DataSet dataSet = createDataSet(new String[]{"1", "name1", "25", "extra"});
+        String tableName = "table_name";
+        createTableWithData(tableName);
+        databaseManager.connect(dbName, USER, PASSWORD);
+        boolean actual = databaseManager.insert(tableName, dataSet);
+        executeQuery(String.format("DROP TABLE IF EXISTS %s", tableName));
+        assertFalse(actual);
+    }
+
+    @Test
+    public void testInsertWhenTypeMismatch() throws SQLException {
```

Comment:
I like this type of test, maybe I'll add this one to my code

Reply:
I also add this type of test to my code )